src/queries/waterTemperature.js

```diff
@@ -0,0 +1,12 @@
+import { waterTemperatureEndpoint } from '../config';
```

Comment:
it seems like the export is missing for this?

Reply:
i cannot see any widget. maybe this is the reason?

Reply:
Yeah, I'll send this to you.